feature-group-definitions/compression-streams.dist.yml

```diff
@@ -0,0 +1,34 @@
+# Generated from: compression-streams.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: Compression streams
+spec: https://wicg.github.io/compression/
+usage_stats:
+  - https://chromestatus.com/metrics/feature/timeline/popularity/3060 # CompressionStream
+  - https://chromestatus.com/metrics/feature/timeline/popularity/3061 # DecompressionStream
+status:
+  baseline: low
+  baseline_low_date: 2023-05-09
+  support:
+    chrome: "103"
```

Comment:
Large parts were supported much earlier, need to review if we should split or override:
https://developer.mozilla.org/en-US/docs/Web/API/CompressionStream#browser_compatibility